h/streamer/websocket.py

```diff
@@ -71,7 +71,7 @@ def handle_message(message):
             jsonschema.validate(payload, filter.SCHEMA)
 
             # Add backend expands for clauses
-            _expand_clauses(payload)
+            _expand_clauses(socket.request, payload)
```

Comment:
@nickstenning Is the `socket.request.db` the same session as the session created in [streamer.py](https://github.com/hypothesis/h/blob/master/h/streamer/streamer.py#L72)?